velox/functions/lib/aggregates/SumCountChecker.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/functions/lib/aggregates/SumCountChecker.h"
+
+namespace facebook::velox::functions::aggregate {
+
+void checkSumCountRowType(TypePtr type, const std::string& errorMessage) {
+  VELOX_CHECK(
+      type->kind() == TypeKind::ROW || type->kind() == TypeKind::VARBINARY,
```

Comment:
type->isRow(), type->isVarbinary() for readability

Should these be USER checks?